MyWebPage/index.html

```diff
@@ -88,5 +88,7 @@ <h1 class="title">Game Cards</h1>
   </div>
 </div>
 </body>
-
+<footer>
+  <p>&copy; Sebastian Morales - 2022</p>
+</footer>
```

Comment:
All the content of the page should be into the `<body>` tag, Including the footer.